yt_dlp/extractor/radiocomercial.py

```diff
@@ -0,0 +1,142 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import (
+    OnDemandPagedList,
+    extract_attributes,
+    get_element_by_class,
+    get_element_html_by_class,
+    get_element_text_and_html_by_tag,
+    get_elements_html_by_class,
+    int_or_none,
+    join_nonempty,
+    try_call,
+    unified_strdate,
+)
+from ..utils.traversal import traverse_obj
+
+
+class RadioComercialIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radiocomercial\.pt/podcasts/[^/]+/\D*(?P<season>\d+)/(?P<id>[\w-]+)'
+    _TESTS = [
```

Comment:
Oh sorry, I should have been clearer on this. I didn't mean that you should indent the tests one more, but just to add the missing trailing commas and combine some of the lines. Take a look at any of the recently added extractors to see how tests are usually formatted ([example](https://github.com/yt-dlp/yt-dlp/pull/7616/files)).

Reply:
No worries. The confusion was on my side :)